pages/Favorites/index.js

```diff
@@ -1,32 +1,48 @@
 import PreviewPicture from "../../components/PreviewPicture";
-import { Container } from "../../components/Container";
+import { Container } from "../../components/styledComponents/Container";
 import { useAtom } from "jotai";
 import { isFavorit } from "../../store/isFavorit";
-import { StyledButton } from "@/components/StyledButton";
+import { StyledButton } from "@/components/styledComponents/StyledButton";
 import React from "react";
 
+async function decreaseFavtoMongoDB(picSRCCloudinary, picID) {
+  await fetch(`/api/Favorites/alltimeFavorites/decreaseFavorites/${picID}`, {
+    method: "PUT",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({ picSRCCloudinary }),
+  });
+
+  return;
+}
+
```

Comment:
I have the impression that I saw this function just shortly before 🤔
This would be a candidate for an externalised function when it´s used in more than one place.